tests/features/steps/wrappers.py

```diff
@@ -4,9 +4,11 @@
 import re
 
 
-def expect_exact(context, expected, timeout):
+def expect_exact(context, expected, timeout, ignore_before=False):
     try:
         context.cli.expect_exact(expected, timeout=timeout)
+        if not ignore_before:
```

Comment:
The `ignore_before` part seems like an unnecessary complication. Why do we need to assert there was nothing printed before the matched pattern?